uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt

```diff
@@ -98,8 +101,8 @@ fun AppContent(
                 )
             }
             composable(
-                route = "other/{otherTab}",
-                deepLinks = listOf(navDeepLink { uriPattern = "$deepLinkUri/other/{otherTab}" }),
+                route = "$OTHER_PATH{otherTab}",
+                deepLinks = listOf(navDeepLink { uriPattern = "$deepLinkUri/$OTHER_PATH{otherTab}" }),
```

Comment:
Thank you for cleaning!
Almost LGTM 👍 
Can you add a new line here?
Our CI style check tool fails. 👀

Reply:
overlooked linter advice. thanks!